torch/csrc/ExceptionsStub.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <exception>
+#include <stdexcept>
+#include <string>
+
+namespace torch {
+
+struct PyTorchError : public std::exception {
```

Comment:
I'd rather avoid adding putting code that's so tightly bound to the original implementation in a different header. Let's put the ifdefs or stubs in the original one

Reply:
Done